<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SerialPortIcons.qrc">
    <normaloff>:/res/images/serial_port_icon.png</normaloff>:/res/images/serial_port_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad_Sequence"/>
    <addaction name="actionSave_Sequence"/>
    <addaction name="actionSave_Sequence_As"/>
    <addaction name="actionView_Log"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_Global_Values"/>
    <addaction name="actionKeep_Edits_Automatically"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionView_Log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View &amp;Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionLoad_Sequence">
   <property name="text">
    <string>&amp;Open Sequence</string>
   </property>
   <property name="toolTip">
    <string>Load a perviously saved sequence from a *.SER or *.SERVO file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Sequence">
   <property name="text">
    <string>Save Sequence</string>
   </property>
   <property name="toolTip">
    <string>Save current sequence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Sequence_As">
   <property name="text">
    <string>Save Sequence As</string>
   </property>
  </action>
  <action name="actionSet_Global_Values">
   <property name="text">
    <string>Set Global Values</string>
   </property>
  </action>
  <action name="actionKeep_Edits_Automatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep Edits Automatically</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SerialPortIcons.qrc"/>
 </resources>
 <connections/>
</ui>
